pkg/controllers/report/background/controller.go

```diff
@@ -377,3 +402,51 @@ func (c *controller) reconcile(ctx context.Context, log logr.Logger, key, namesp
 	}
 	return nil
 }
+
+func (c *controller)reconcileVapReports(
```

Comment:
This function isn't compiled. we don't need it.

Reply:
I was thinking about the same. Rather to apply the logic in reconcileReports. But this function has the backgroundpolicies of type v1alpha1.ValidatingAdmissionPolicy and creates a scanner for the VAP.